.travis.yml

```diff
@@ -61,6 +66,8 @@ script:
       PX4_DOCKER_REPO=px4io/px4-dev-clang ./Tools/docker_run.sh 'make clang-tidy-parallel';
     elif [ "${TRAVIS_OS_NAME}" = "osx" ]; then
       make tests;
+    elif [[ "${BUILD_TARGET}" = "px4_clang" && "${TRAVIS_BRANCH}" != "coverity" ]]; then
+      PX4_DOCKER_REPO=px4io/px4-dev-nuttx-clang ./Tools/docker_run.sh 'make px4fmu-v2_default';
```

Comment:
The tag is the date at the end and corresponds to git tags here - https://github.com/PX4/containers
That isolates the builds in PX4 from changing containers.